src/activity/activity.jsx

```diff
@@ -0,0 +1,123 @@
+import { h, Component } from 'preact'
+import cx from 'classnames'
+
+import _max from 'lodash/max'
+import _mean from 'lodash/mean'
+
+import { mountComponent, chunk } from 'utils'
+
+import { FACETS, GRANULARITIES } from 'src/constants'
+
+import Tabs from './components/tabs'
+import Dropdown from './components/dropdown'
+import Histogram from './components/histogram'
+
+import styles from './activity.scss'
+
+import histogramUsers from '../../public/mocks/histogram-users.json'
+import histogramFeatures from '../../public/mocks/histogram-features.json'
+
+class DailyActivity extends Component {
+  constructor (props) {
+    super(props)
+    this.updateFacet = this.updateFacet.bind(this)
+    this.updateGranularity = this.updateGranularity.bind(this)
+    this.state = this.formatState(props.data)
+  }
+
+  formatState (data) {
+    return {
+      granularity: 'd',
+      facet: 'u',
+      Users: this.formatUsers(data),
+      Features: this.formatFeatures(data)
+    }
+  }
+
+  formatUsers (data) {
+    return histogramUsers // .slice(0, 6)
```

Comment:
In the end you are not using the mock data from buildings_country_hti.json at all? Whatever the final mock file is (we should switch to real data anyways), the fact that we don't use data in this method should be picked up by the linter IMHO.